proto/WORKSPACE.bazel

```diff
@@ -1,3 +1,22 @@
+##### Protobuf Rules for Bazel
```

Comment:
Just a single "#" for comment, e.g., "# Protobuf Rules for Bazel."

Reply:
Done